### PR TITLE
[Docs] Swap sentences to comply with evaluation logic

### PR DESCRIPTION
**Pull request by mjuchli-da**
Statements generally are evaluated by preorder traversal. Thus the second and third (last) consequences need to be swapped (as they otherwise would follow level-order traversal).

### CODE REVIEW

File: docs/source/concepts/ledger-model/ledger-integrity.rst [hunk 1]
```diff
@@ -490,16 +490,13 @@ on this exercise, so this complies with the authorization requirement.
...
-consequence is the creation of the paint agreement, which has Alice
-and the painter as signatories. Since they both authorize this action,
-this is also permissible. Finally, the creation of the new IOU (for P)
-is a consequence of the exercise on the old one (for A). As the
-old IOU was signed by the bank, and as Alice was the actor of
-the exercise, the bank and Alice jointly authorize the
-creation of the new IOU. Since the bank is the sole signatory of this
-IOU, this action is also permissible. Thus, the entire third commit
-is also well-authorized, and then so is the ledger.
+Alice as the actor; so this is permissible.
```
Comment (carrielaben-da) [inline docs/source/concepts/ledger-model/ledger-integrity.rst:493]:
```suggestion
Alice as the actor, so this is permissible. 
```

File: docs/source/concepts/ledger-model/ledger-integrity.rst [hunk 2]
```diff
@@ -490,16 +490,13 @@ on this exercise, so this complies with the authorization requirement.
...
-old IOU was signed by the bank, and as Alice was the actor of
-the exercise, the bank and Alice jointly authorize the
-creation of the new IOU. Since the bank is the sole signatory of this
-IOU, this action is also permissible. Thus, the entire third commit
-is also well-authorized, and then so is the ledger.
+Alice as the actor; so this is permissible. 
+The second consequence is the creation of the new IOU (for P) by exercising the old IOU (for A).
+As the IOU was formerly signed by the bank, with Alice as the actor of the exercise, they jointly authorize this creation.
+This action is permissible as the bank is the sole signatory.
+The final consequence is creating the paint agreement, with Alice and the painter as signatories.
```
Comment (carrielaben-da) [inline docs/source/concepts/ledger-model/ledger-integrity.rst:497]:
```suggestion
The final consequence is creating the paint agreement with Alice and the painter as signatories.
```

**Review by carrielaben-da — APPROVED** [review]
A couple minor punctuation tweaks.

Since the order matters in this paragraph, we should also consider presenting the information as a numbered list - this may involve broader changes to the overall document so we can go ahead and merge this and then rework.

**Comment by mjuchli-da** [comment]
> A couple minor punctuation tweaks.

Suggestions are committed.

> Since the order matters in this paragraph, we should also consider presenting the information as a numbered list - this may involve broader changes to the overall document so we can go ahead and merge this and then rework.

Agreed, a numbered list would make it easier to follow.

**Pull request merged**